### DISCUSSION TITLE
zh/faq/why

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#44] [Announcements]
# zh/faq/why

AList v3的文档

https://alist.nn.ci/zh/faq/why.html

**Comment by wj742421292** [comment]
添加了3个移动云盘后, 重新登录, 先添加的2个提示令牌失效, 请问是什么原因

**Comment by at86397388** [comment]
为什么我在阿里云盘转存了视频，通过alist里却看不到，一定要在当前目录点右边的刷新才会出来？

**Reply by at86397388** [reply]
不刷新的话，一直都不会出来……

**Reply by Hanxu4131** [reply]
我的大概5-10分钟就会出来

**Comment by at86397388** [comment]
页面能自动刷新么？？

**Comment by liujixu1984** [comment]
大家 不使用https功能能不能拥有打包下载功能啊

**Comment by Helsing9527** [comment]
Docker部署遇见这个问题怎么解决呀
```tex
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "./alist": stat ./alist: no such file or directory: unknown.
```

**Comment by qianku** [comment]
我安装的V3.3.0版本。使用update命令更新提示：出错了，V3与V2不兼容，请先卸载V2或更换位置安装V3
这是为啥？

**Comment by JokingLove** [comment]
为什么我用docker部署之后，admin 登录，密码是从日志中找的，一直提示密码错误呢

**Comment by FashionCrazy** [comment]
Failed init storage: ocr error:
添加天翼云盘之后显示的

**Comment by mtk2025** [comment]
ipa安装时没有图标怎么操作？plist文件图标链接不能自定义吗？

**Comment by ahhszhcsssCN** [comment]
115的错误，这是啥意思。。
{"state":1,"code":40101033,"message":"\u8be5\u8d26\u53f7\u4e8e18:04\u5728115\u6d4f\u89c8\u5668(Windows)\u767b\u5f55\u3002\u5982\u975e\u672c\u4eba\u64cd\u4f5c\uff0c\u8bf7\u53ca\u65f6\u4fee\u6539\u5bc6\u7801\uff01","data":{"gray_theme":{"30":{"status":3},"31":{"status":3},"32":{"status":3},"33":{"status":3}}},"error":"\u8be5\u8d26\u53f7\u4e8e18:04\u5728115\u6d4f\u89c8\u5668(Windows)\u767b\u5f55\u3002\u5982\u975e\u672c\u4eba\u64cd\u4f5c\uff0c\u8bf7\u53ca\u65f6\u4fee\u6539\u5bc6\u7801\uff01","errno":40101033}: unexpected error

**Reply by nanvon** [reply]
Unicode 转中文：该账号于18:04在115浏览器(windows)登录。如非本人操作，请及时修改密码！

**Comment by SourBean** [comment]
本地存储上传文件有1G大小的限制，请问在哪里可以修改参数

**Comment by chazchi** [comment]
webdav挂载阿里云盘后，下载提示权限不足

**Reply by zhouzhe-one** [reply]
同样问题求解

**Comment by sunwu57** [comment]
failed get objs: failed to list objs: invalid X-Device-Id     阿里云盘报错

**Comment by caesaranthony** [comment]
请教一下，我添加移动家庭云盘，在保持“根文件夹ID”为空的情况下，打开文件夹，显示不出云盘里的文件，在填入“根文件夹ID”后，显示“failed get objs: failed to list objs: 鉴权失败”，麻烦帮忙看看是什么原因，谢谢。

**Comment by dpps1228** [comment]
怎么加入百度网盘啊

**Comment by iorangy** [comment]
发送到Aria2时，提示“Failed to send to aria2: AxiosError: Network Error”，请问是什么原因呢？

**Comment by friedrica137** [comment]
手动安装以及桌面版均无法登入
win7设备，手动安装alist-windows-amd64.zip，首次运行输入alist server后出现初始用户名及密码，但start server@0.0.0.0:5244后出现qbittorrent not ready以及Aria2 not ready。首次登录http://ip:5244可登入，并修改初始用户名及密码，其后点击管理时显示http://ip:5244拒绝访问，并再未成功登入。
手动安装alist-windows-386.zip问题同上。
桌面版安装过程中显示there is a problem with this Windows Installer package. A program run as part of the setup did not finish as expected.然后alist-desktop Setup Wizard会提示说ended prematurely because of an error
希望问题能够得到解决。

**Comment by taskschd-E5** [comment]
挂载Cloudreve时提示Key: 'Storage.MountPath' Error:Field validation for 'MountPath' failed on the 'required' tag
用WebDAV挂载也是Key: 'Storage.MountPath' Error:Field validation for 'MountPath' failed on the 'required' tag

**Reply by taskschd-E5** [reply]
当然我挂载成功了1次，ZYB14一次都没成功。。

**Comment by Alibaba404** [comment]
使用Chrome ,Edge 浏览器无法进行打包下载,/ping 接口返回 You need to enable JavaScript to run this app.    但是Safari就可以 我使用的是本地存储

**Comment by x3fang** [comment]
为什么网站在自定义头部 和 自定义内容里添加到内容，保存后显示乱码

**Comment by sa2yuan** [comment]
百度网盘出现了
failed get objs: failed to list objs: req: [https://pan.baidu.com/rest/2.0/xpan/file] ,errno: -9, refer to https://pan.baidu.com/union/doc/
是啥情况

**Comment by zhouzhe-one** [comment]
挂载阿里云盘open后，无法进行资源下载，错误提示：Failed to sync with 43 errors: last error was: failed to open source object: 403 Forbidden

**Comment by bn-yt** [comment]
我在pc桌面版将云盘挂载到Alist后，又根目录挂载到本地，就在本地看到一个1PB的网络硬盘，里面可以直接访问云盘文件。请问这个1PB网络硬盘是真实的吗，这个空间是从哪提供的？另一个问题是在远端服务器部署完成后，同样操作却不能在服务器本地访问云盘文件？

**Comment by h770393141** [comment]
为什么复制链接里的地址和下载里的地址不一样？文件是alistV3通过局域地址连另一台机器的alistV3，复制地址是公网地址，下载缺是局域网的地址，这个下载能设置成公网地址吗

**Comment by Eneryg** [comment]
alist挂载阿里云盘，播放视频时在16：59分就会卡住，只有重新刷新网页才能继续播放

**Reply by Eneryg** [reply]
刷新之后在32：59秒又卡顿，基本上就是播放视频16分钟左右就会卡顿，只能重新刷新才能继续看

**Comment by summer0607** [comment]
为什么我右键复制没有出现复制直链选项，只有一个复制链接，粘贴后是http://192.168.31.22:5244/%E8%93%9D%E5%A5%8F%E4%BA%91/pj%E8%BD%AF%E4%BB%B6这样的链接。。。

**Comment by haohao232323** [comment]
上传出现Cannot read properties of undefined (reading 'status')错误提示，使用了分卷压缩，同批分卷的大部分上传成功，个别出现这个错误，不知道是哪个环节的问题。

**Comment by cancundeyingzi** [comment]
能不能自动点击添加为任务

**Comment by FYboss** [comment]
在alist页面离线下载，提示success，但在aria2后台显示下载失败，并且在群晖前台手动添加一样的链接是可以下载成功的；alsit后台我已经添加了aria2的密钥和地址，也提示aria2功能正常，但就是无法离线下载，夸克、阿里云盘、本地存储都不行
